packages/realm/README.md

```diff
@@ -30,24 +40,24 @@ If you want to contribute to any of our packages, you are welcome to do so. We w
 
 In tier 1 we will respond to issues in a timely manner during workdays from CET timezone, and we will work on bug fixing and adding new features.
 
-* [Realm JavaScript](https://www.npmjs.com/package/realm) (NPM tag: `latest`) on node.js (LTS) and Electron on Windows, MacOS, and Linux
-* [Realm JavaScript](https://www.npmjs.com/package/realm) (NPM tag: `latest`) with the latest React Native version 0.71.0 on Android and iOS
-* [@realm/react](https://www.npmjs.com/package/@realm/react) (NPM tag: `latest`) in conjunction with the latest [Realm JavaScript](https://www.npmjs.com/package/realm) release
+* [`realm`](https://www.npmjs.com/package/realm) (NPM tag: `latest`) on node.js (active LTS) and Electron on Windows, MacOS, and Linux
+* [`realm`](https://www.npmjs.com/package/realm) (NPM tag: `latest`) with the latest React Native version 0.71.0 on Android and iOS
+* [`@realm/react`](https://www.npmjs.com/package/@realm/react) (NPM tag: `latest`) in conjunction with the latest [`realm`](https://www.npmjs.com/package/realm) release
 
 ### Tier 2 - best effort
 
 Some packages are considered to be mature and stable, and we will support them as good as we can when time permits.
 
-* [Realm Web](https://www.npmjs.com/package/realm-web) (NPM tag: `latest`)
+* [`realm-web`](https://www.npmjs.com/package/realm-web) (NPM tag: `latest`)
 * [Realm Studio](https://github.com/realm/realm-studio) ([latest](https://github.com/realm/realm-studio/releases/latest) release) on Windows, MacOS, and Linux
-* Any other release of [Realm JavaScript](https://www.npmjs.com/package/realm) not covered by tier 1
+* Any other release of [`realm`](https://www.npmjs.com/package/realm) not covered by tier 1
 
 ### Tier 3 - experimental
 
 The third tier covers our experimental packages. We work on them occasionally, and they are likely to change radically when we do.
 
-* [Realm Babel plugin](https://www.npmjs.com/package/@realm/babel-plugin)
-* [Realm Flipper plugins](https://www.npmjs.com/package/realm-flipper-plugin)
+* [`@realm/babel-plugin`](https://www.npmjs.com/package/@realm/babel-plugin)
+* [`realm-flipper-plugin`](https://www.npmjs.com/package/realm-flipper-plugin)
```

Comment:
I think we should remove it

Reply:
```suggestion
```

Reply:
I'll leave actually removing it from the repository, to a separate PR 🙂